fire/helputils.py

```diff
@@ -126,7 +126,8 @@ def HelpString(component, trace=None, verbose=False):
   return '\n'.join(lines)
 
 
-def _UsageStringFromFnDetails(command, args, varargs, keywords, defaults):
+def _UsageStringFromFnDetails(command, args, varargs, varkw, defaults,
```

Comment:
Let's just have this function accept an argspec instead of a long list of parameters.
And as long as we're here let's rename the function so that we have consistent terminology.

"def _UsageStringFromArgSpec(command, argspec):"

(This function is only used in one place, so no need to go searching for references.)

(I'm actually inclined to rename all our uses of the word "argspec" to "signature", but we can do that separately from this fix.)